### DISCUSSION TITLE
Cursor id unsupported

### DISCUSSION THREAD

**Discussion by Rutledge** [uriyyo/fastapi-pagination#1355] [Q&A] [answered]
It seems like cursor id pagination is currently unsupported?

https://app.stainlessapi.com/docs/guides/configure#pagination-type

Or is this a modified version of cursor pagination?

**Comment by uriyyo** [comment, marked as answer]
@Rutledge It's kinda subset of cursor pagination, and it should be supported by sqlachemy extension
